features/task_execution.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Task execution in BQFlow
```

Comment:
This example is overly simple and brittle, this pattern will not support actual JSON API calls and SQL logic without significant rewrites or translating JSON to YAML.

BigQuery returns JSON not YAML rows.

Reply:
Can you expand upon this? I'm not clear on which pattern you're talking about. I did run this with the expanded search/replace json -> yaml and found that it worked as-is.

YAML is a superset of JSON, so any JSON is interpreted just fine as YAML.